doc/UsersGuide/source/conf.py

```diff
@@ -41,6 +41,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
```

Comment:
@jdkublnick In line 27, please change to `version = 'develop'`
and in line 29, change to `release = 'Develop Branch Documentation'`